Demo/src/main/java/com/braintreepayments/demo/LocalPaymentsActivity.java

```diff
@@ -40,7 +40,7 @@ protected void onAuthorizationFetched() {
         }
 
         try {
-            mBraintreeFragment = BraintreeFragment.newInstance(this, Settings.getLocalPaymentsTokenizationKey(this));
+            mBraintreeFragment = BraintreeFragment.newInstance(this, "sandbox_f252zhq7_hh4cpc39zq4rgjcg");
```

Comment:
👀 Should this have been replaced?

Reply:
Yes, the tokenization key being used previously did not have local payment methods enabled so we could not test this flow at all in our demo app. The previous key is used across several internal apps, so updating it in our demo app allows us to test the LPM flow as expected. Additionally the new key is the same as what we use for LPM on iOS so allows for consistency across platforms.